src/components/Speciality/Intro.js

```diff
@@ -41,14 +41,14 @@ const IntroSection = ({ speciality }) => (
                 {speciality.title}
               </SectionTitle>
               <BodyPrimary reverse muted>
-                {speciality.seoText.content[0].content[0].value}
+                {speciality.seoText}
               </BodyPrimary>
             </Flex>
           </Col>
           <Col width={[1, 1, 1, 1, 6 / 12]}>
             <img
-              alt={speciality.introGraphic.title}
-              src={speciality.introGraphic.file.url}
+              alt={speciality.introGraphicTitle}
+              src={speciality.introGraphicFileUrl}
               style={{ maxHeight: '100%' }}
```

Comment:
Whilst we're here, lets update this intro section to use RatioContainer with the width and height set to `100`

Also, go back to the query and update it to use `GatsbyContentfulFluid_withWebp` not the `file {  title, url }`. 
```
fluid(maxWidth: 30) {
    ...GatsbyContentfulFluid_withWebp
}
```

Reply:
I've pushed this change, it doesn't look to be breaking anything...